components/SoftwareAccordion.tsx

```diff
@@ -26,13 +26,13 @@ const SoftwareAccordionItem = ({
     versions: ZkvmVersion[]
     vendor: Vendor
     totalClusters: number
-    zkvmClusters: number
+    activeClusters: number
   }
 }) => (
   <AccordionItem value={value} className="col-span-5 grid grid-cols-subgrid">
     <div className="col-span-5 grid grid-cols-subgrid items-center gap-12 border-b hover:bg-primary/5 dark:hover:bg-primary/10">
       <div className="col-start-1 flex items-center gap-3">
-        <Link href="/zkvm/#TODO-zkvm-id" className="hover:underline">
+        <Link href={`/zkvms/${zkvm.id}`} className="hover:underline">
```

Comment:
Path naming currently set up where plural is the list (`/zkvms`) and singular with id is for details (`/zkvm/id`)

```suggestion
        <Link href={`/zkvm/${zkvm.id}`} className="hover:underline">
```

(Same with /provers /prover/id, /blocks /block/id /clusters /cluster/id)